packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -104,21 +104,78 @@ export type GetSelector<Component extends (props: any) => JSX.Element | undefine
   : undefined;
 
 /**
- * Get the selector for a specified component.
+ * Get the selector for {@link VideoGallery}.
  *
- * Useful when implementing a custom component that utilizes the providers
- * exported from this library.
+ * @public
+ */
+export function getSelector<T>(component: typeof VideoGallery): VideoGallerySelector;
```

Comment:
Should be `<T extends typeof VideoGallery>` ensures that this function overload is not picked for any other component.